spec/requests/cinema_halls_spec.rb

```diff
@@ -0,0 +1,43 @@
+require "rails_helper"
+
+RSpec.describe "CinemaHalls requests" do
+
+  describe "GET /cinema_halls" do
+    let!(:cinema_hall) { CinemaHall.create(hall_name: "Some name", volume: 500) }
```

Comment:
You added the `cinema_hall` factory but didn't use it. You can change the above to:
```
let!(:cinema_hall) { create :cinema_hall, name: "Some name", volume: 500 }
```